### PR TITLE
Port 3: Port another 14 files

### PR DESCRIPTION
**Pull request by hahakiki2010**
Porting to Python 3 compatible with Python2, and remove
autotest dependencies

Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

**Comment by hahakiki2010** [comment]
[root@localhost src]# avocado run --vt-type libvirt timer_management.specific_timer.kvmclock.present_no.offset_utc.no_stress.test_specific_timer
JOB ID     : ffb62f514593d7e5a978facbcd38223131905c26
JOB LOG    : /root/avocado/job-results/job-2018-03-22T16.32-ffb62f5/job.log
 (1/1) type_specific.io-github-autotest-libvirt.timer_management.specific_timer.kvmclock.present_no.offset_utc.no_stress.test_specific_timer: PASS (46.29 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 48.82 s
JOB HTML   : /root/avocado/job-results/job-2018-03-22T16.32-ffb62f5/results.html

File: libvirt/tests/src/hotplug_serial.py [hunk 1]
```diff
@@ -170,8 +167,8 @@ def confirm_hotplug_result(char_dev, id=0):
             session.cmd("dd if=/tmp/file of=%s &" % serial_file)
             dev_file = "/dev/pts/%s" % id
             if not os.path.exists(dev_file):
-                raise error.TestFail("%s doesn't exist." % dev_file)
-            p = subprocess.Popen(["/usr/bin/cat", dev_file],
+                test.fail("%s doesn't exist." % dev_file)
+            p = subprocess.Popen(["/usr/bin/cat", dev_file], universal_newlines=True,
```
Comment (chunfuwen) [inline libvirt/tests/src/hotplug_serial.py]: may set shell=True

File: libvirt/tests/src/multifunction.py [hunk 2]
```diff
@@ -171,7 +170,8 @@ def attach_additional_device(vm_name, disksize, targetdev, params):
     logging.info("Attaching disk...")
     disk_path = os.path.join(data_dir.get_tmp_dir(), targetdev)
     cmd = "qemu-img create %s %s" % (disk_path, disksize)
-    status, output = commands.getstatusoutput(cmd)
+    ret = process.run(cmd)
```
Comment (chunfuwen) [inline libvirt/tests/src/multifunction.py]: getstatusoutput will redirect std.err to std.out,
you may use process.getstatusoutput,see https://github.com/avocado-framework/avocado/pull/254

File: libvirt/tests/src/hotplug_serial.py (hunk 1 above)
Comment (hahakiki2010) [reply libvirt/tests/src/hotplug_serial.py]: fixup

File: libvirt/tests/src/multifunction.py (hunk 2 above)
Comment (chunfuwen) [reply libvirt/tests/src/multifunction.py]: OK, just go ahead

**Pull request merged**